docs/rackhd/event_notification.rst

```diff
@@ -299,4 +303,50 @@ An example of multiple filters
 Web Hook APIs
 ^^^^^^^^^^^^^^^
 
-**TODO** (The APIs of web hooks developments are ongoing, this section wiil be updated when they are finished).
+
+**Create a new hook**
+
+
+.. code-block:: REST
+
+    POST /api/2.0/hooks
+    {
+        "url": "http://www.abc.com/def"
+    }
+
+
+**Delete an existing hook**
+
+
+.. code-block:: REST
+
+    DELETE /api/2.0/hooks/:id
+
+
+**Get a list of hooks**
+
+
+.. code-block:: REST
+
+    GET /api/2.0/hooks
+
+
+**Get details of a single hook**
+
+
+.. code-block:: REST
+
+    GET /api/2.0/hooks/:id
+
+
+**Update an existing hook**
+
+
+.. code-block:: REST
+
+    PATCH /api/2.0/hooks/:id
+    {
+        "id": "21ea5fc7-ec17-4a83-9536-2733d6ce0ede",
```

Comment:
The `id` has already been specified in URL, so remove the `id` in payload (or changing to `url`), otherwise, it will confuse the reader.